README.md

```diff
@@ -1,3 +1,41 @@
 # Vaadin React components
 
 React wrappers for Vaadin components.
+
+## Using Local React Components in a Vaadin Project
+
+To configure your Vaadin project to import React components from a local repository, add this Vite plugin to `vite.config.ts`:
```

Comment:
Why not add the plugin to `vite.config.ts` in this PR?

Reply:
I'm not sure what you mean... This plugin is intended to be used in other projects, not react-components itself.

Reply:
Ah, got it. Misunderstood the purpose.

Reply:
I updated the text to clarify this.